docs/self-managed/components/orchestration-cluster/identity/connect-external-identity-provider.md

```diff
@@ -0,0 +1,437 @@
+Claims from the token are used to identify the user's username and, optionally, groups for easier assignment and management.
+In addition, Mapping Rules can be used to map token claims to Camunda roles, authorizations, or tenants.
+
+Note that user information is not stored in the Orchestration Cluster when using OIDC. Hence you will notice that features like user search, user validation on assigning authorizations or tenants and user management are not available in the Orchestration Cluster Identity UI when using OIDC.
+
+For information on configuring User Interface authentication, see the [User Interface](#user-interface) section below.
+
+**M2M API access for Connectors and Workers:**
+
+Connectors, job workers or other implementations that use the Orchestration Cluster REST or gRPC APIs (e.g. by using one of the Camunda Clients) do not use the browser-based authorization code flow. Instead, they authenticate using client credentials. The authentication method for Camunda Clients is separate from the interactive user login and typically require additional configuration in your Identity Provider and Camunda.
+
+For information on configuring Connectors, job workers, or other applications to authenticate with the Orchestration Cluster APIs using client credentials, see the [M2M API access](#m2m-api-access) section below.
+
+## User Interface
+
+### Goal of this guide
+
+By the end of this guide your users should be able to authenticate to the Camunda 8 Orchestration Cluster using your own Identity Provider.
+
+### Prerequisites
+
+- Camunda 8 Orchestration Cluster (Self-Managed)
+- Access to an OIDC-compliant Identity Provider (e.g., Keycloak, Auth0, Okta, Microsoft EntraID)
+- Client credentials (client ID, client secret, issuer URI) from your Identity Provider
+
+### Step 1: Prepare Your Identity Provider
+
+Before configuring Camunda, perform these high-level steps in your Identity Provider:
+
+1. **Register a new application/client** for Camunda 8 Orchestration Cluster in your Identity Provider.
+   - Set the application type to "Web" or "Confidential".
+   - Enable OIDC support.
+   - Configure the necessary scopes (e.g., `openid`, `profile`, `email`).
+   - Ensure the client is allowed to access user information.
+2. **Set the redirect URI** to match your Camunda deployment (default: `http://localhost:8080/sso-callback`).
+3. **Assign users or groups** who should have access to Camunda 8.
+4. **(Optional) Configure group or other claims** if you want to use group-based authorizations or mapping rules in Camunda.
+5. **Note the client ID, client secret, and issuer URI** for use in Camunda configuration.
+
+> For most Identity Providers, the default claim for the username is `sub` (subject). If you want to use a different claim (e.g., `preferred_username` or `email`), configure your Identity Provider to include it in the token and update the Camunda configuration accordingly.
```

Comment:
💭 Can we link to the configuration docs here also or note the configuration param

Reply:
good idea, resolved